admin/admin.go

```diff
@@ -58,6 +79,38 @@ func (err *JSONRPCError) ErrorCode() int {
 	return err.Code
 }
 
+type BigIntStr big.Int
```

Comment:
I think we can use go-ethereum for this

https://github.com/ethereum/go-ethereum/blob/master/common/math/big.go#L43

Reply:
oh nice, that's exactly what I need